keps/sig-api-machinery/2161-apiserver-default-labels/README.md

```diff
@@ -408,6 +369,8 @@ makes it feel special." - @thockin
 
 ### Add new select-by-name capabilities to policy APIs
 
+* Add an 'in memory' virtual label that is defaulted by the apiserver but not stored in etcd
+* Add a new syntax for selectors on labels that allow them to dynamically select things in fields
```

Comment:
Can you add reasons for / against? Like I'm not sure this last one is actually worse than the currently selected design.

Reply:
APIs are versioned.  Are we going to spec an API version in that syntax?  Are we going to auto-convert that field between API version?

This is the same problem that lead us to leave the various field-selector implementations a) hardcoded and b) VERY restricted.